Packs/RSANetWitness_v11_1/ReleaseNotes/3_2_3.md

```diff
@@ -0,0 +1,3 @@
+##### RSANetWitness_v11_1
+
+- Started adoption process.
```

Comment:
```suggestion
#### Integrations
##### RSANetWitness_v11_1
- Started adoption process.
```

Reply:
Hi @yucohen I have updated the changes. Also we (Netwitess) will also be adopting the Packets and logs pack as well. PLease have a look

Reply:
@adit-ghildiyal thanks! 
I see that the validations are still failing, but I'll merge and fix it internally :)